frontend/src/features/properties/filter/PropertyFilter.test.tsx

```diff
@@ -24,13 +26,76 @@ jest.mock('actionCreators/propertyActionCreator');
 (fetchPropertyNames as any).mockImplementation(jest.fn(() => () => ['test']));
 
 const mockedAxios = axios as jest.Mocked<typeof axios>;
+const mockKeycloak = (claims: string[]) => {
```

Comment:
We want this in a shared function, but not strictly required for this PR.

Reply:
I think I extracted it from some other test and modified for my use-case.  I'd prefer a shared function too, but at this point I'm leaning toward adding technical debt, as I don't have enough hours to complete critical work.